xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -477,7 +477,8 @@ final class DatabaseService {
             return EntryStub(
                 address: address,
                 excerpt: excerpt,
-                modified: modified
+                modified: modified,
+                author: nil
```

Comment:
Nit: seems unfortunate we can't get the author info from the DB. DID and petname are technically in there.

Non-blocking.

Reply:
Yah. We could build a DB cache for the profile information and then retrieve it as part of queries like this. It would simplify a large part of the loading process.

Tracked in https://github.com/subconsciousnetwork/subconscious/issues/813